gitsome/utils.py

```diff
@@ -184,3 +184,14 @@ def _safe_split(self, text):
             return words
         except:
             return text
+
+    def sanitize_if_none(self, text):
+        """Sanitizes text to ensure it is not None
```

Comment:
Could you [PEP-257-ify](https://www.python.org/dev/peps/pep-0257/) this docstring to the following?

Sanitize text to ensure it is not None.